### DISCUSSION TITLE
Vote Rewards

### DISCUSSION THREAD

**Discussion by thedarkonekiller** [Zrips/CMI#6864] [Enhancement]
Hello,
have the possibility to randomize the voting rewards, add the parameters:
Randomize: true/false and DuplicateRandomize: true/false
As in schedules.yml

**Comment by mrfloris** [comment]
I resolved this myself by making a few kits, with numbers.. and then using the `/cmi kit votereward%the cmi random placeholder here% [playerName]` method. 

default group can use random 1 to 4
vip group random 5 to 10
